src/presentation/ui/component/databases/databaseUserPrivilegesRadioInputsSwitchToggles.templ

```diff
@@ -0,0 +1,32 @@
+package componentDatabases
+
+import (
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	"strings"
+)
+
+var availableDatabaseUserPrivileges = []string{
```

Comment:
This should be coming from the value object, those are business rules.

Reply:
Done!